libvast/vast/concept/parseable/vast/si.hpp

```diff
@@ -55,6 +56,7 @@ namespace parsers {
 
 static auto const count = si_parser<vast::count>{};
 static auto const integer = si_parser<vast::integer>{};
+static auto const bytesize = count >> ~ch<'B'>;
```

Comment:
Just to make sure: is the intention that we *both* support `42k` and `42kB`?

Reply:
Yes, to be consistent with other command-line tools (e.g. `dd`), and also because there's no ambiguity so it doesn't hurt to accept generously.